README.md

```diff
@@ -114,3 +114,20 @@ The `Package` class should provide access to the full dependency tree.
 ```sh
 git submodule init
 ```
+
+
+## Registry URI 
+
+The URI to lookup a package from a registry should follow the following format. (subject to change as the Registry Contract Standard makes it's way through the EIP process)
+
+```
+scheme://authority/package-name/version
```

Comment:
thoughts on some alternate formats which I don't necessarily think are better but worth exploring other options.

- `scheme://package-name@authority/version`
- `scheme://package-name@authority?version=version` (less easy to type out by hand because of query parameter escaping)
- `scheme://authority/package-name?version=version` (less easy to type out by hand because of query parameter escaping)

In all three of these examples, the distinction between `package-name` and `version` is clearer as opposed to having to do string manipulation to separate them (as they are in distinctly different parts of the url scheme).

> I think in all of these, version should be optional, if left off, the *latest* version would be used.